docker-compose.yml

```diff
@@ -1,17 +1,21 @@
-version: '2'
+version: '3'
 
 services:
   master:
     container_name: 'citus_master'
-    image: 'citusdata/citus:6.2.3'
+    image: 'citus:modern'
     ports: ['5432:5432']
     labels: ['com.citusdata.role=Master']
-    volumes: ['/var/run/postgresql']
+    environment:
```

Comment:
These shouldn't be set here; they're included in the image if the user wants them but shouldn't be necessary for us. End-users can extend our image or apply environment variables if they wish to specify these, but our template shouldn't include them.